openedx/core/djangoapps/theming/helpers.py

```diff
@@ -45,18 +37,15 @@ def is_request_in_themed_site():
     This is a proxy function to hide microsite_configuration behind comprehensive theming.
     """
     # We need to give priority to theming/site-configuration over microsites
-    return configuration_helpers.is_site_configuration_enabled() or microsite.is_request_in_microsite()
+    return configuration_helpers.is_site_configuration_enabled()
 
 
 def get_template(uri):
     """
     This is a proxy function to hide microsite_configuration behind comprehensive theming.
     :param uri: uri of the template
     """
-    # We need to give priority to theming over microsites
-    # So, we apply microsite template override only when there is no associated theme,
-    if not current_request_has_associated_site_theme():
-        return microsite.get_template(uri)
+    return None
```

Comment:
We might be able to delete this function after my edxmako PR is merged, it may not be used anywhere else.